.github/workflows/integration-tests.yml

```diff
@@ -162,3 +162,40 @@ jobs:
         with:
           artifact_name: "generator"
           min_bytes: "0"
+
+  idempotency:
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      fail-fast: false
+      matrix:
+        language:
+          - java
+          - csharp
+          - go
+          - typescript
+          - ruby
+          - php
+          - python
+        description:
+          - "./tests/Kiota.Builder.IntegrationTests/InheritingErrors.yaml"
+          - "./tests/Kiota.Builder.IntegrationTests/NoUnderscoresInModel.yaml"
+          - "oas::petstore"
+          - "apisguru::twitter.com:current"
+          - "apisguru::notion.com"
+          - "apisguru::stripe.com"
+          - "https://raw.githubusercontent.com/googlemaps/openapi-specification/main/dist/google-maps-platform-openapi3.yml"
+          - "apisguru::meraki.com"
+          - "https://developers.pipedrive.com/docs/api/v1/openapi.yaml"
+          - "apisguru::twilio.com:api"
+          - "apisguru::docusign.net"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: generator
+          path: publish
+      - run: chmod a+x ./publish/kiota
+      - name: Verify idempotency
```

Comment:
would you mind adding an upload artifacts steps after this one? and maybe only on fails?

Reply:
should be fixed now